HW04.Rmd

```diff
@@ -29,35 +41,81 @@ Knitr is part of the knitr package. Allows you to change column names and add ca
 
 
 ```{r month, warning=FALSE}
-
+monthly_gun_deaths <- gun_deaths %>%
+                        select(month) %>%
+                        mutate(month = factor(month.abb[month],
+                                              levels = c("Jan", "Feb", "Mar", "Apr",                                                         "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"))) %>%
+                        group_by(month) %>%
+                        count()
+
+kable(monthly_gun_deaths, align=rep('c'), format = "markdown",
+      col.names = c("Month", "Total Deaths"),
+      caption = "Number of Gun Deaths per Month")
 ```
 
 
 ### Generating a bar chart with human-readable labels on the x-axis. That is, each month should be labeled "Jan", "Feb", "Mar" (full or abbreviated month names are fine), not `1`, `2`, `3`.
 
 ```{r month_chart}
-
+ggplot(monthly_gun_deaths, aes(month,n)) +
+    geom_col(fill = "maroon") +
+    labs(title = "Number of Gun Deaths per Month in the US (2012-2014)", 
+       x = "Month", 
+       y = "Number of Deaths") +
+    theme(plot.title = element_text(hjust = 0.5))
+    
 ```
 
 ## Generating a bar chart that identifies the number of gun deaths associated with each type of intent cause of death. The bars should be sorted from highest to lowest values.
 
 ```{r intent}
-
+gun_deaths %>%
+    count(intent) %>%
+    ggplot(mapping = aes(reorder(intent, -n), n)) +
+    geom_col(fill = "maroon") +
+    geom_text(aes(label = n), vjust = -0.5, size = 3, color = "grey50") +

```

Comment:
Nice touch adding the exact numbers above each bar, that makes it really clear, easy, and quick to understand.